en/django_start_project/README.md

```diff
@@ -252,6 +252,15 @@ or on Glitch:
 https://name-of-your-glitch-project.glitch.me
 ```
 
+If you are using RunCode cloud platform, the URL will look something like this:
```

Comment:
Same again, if we ever include a line such as "If you're using the RunCode environment" in the existing instructions, we need to preceed the previous instructions with "If you have followed the installation guide" to distinguish upfront that these instructions are not applicable.

Reply:
I have just moved the RunCode instructions to one place so this is no longer necessary. I now just point the readers to the RunCode section.

> Same again, if we ever include a line such as "If you're using the RunCode environment" in the existing instructions, we need to preceed the previous instructions with "If you have followed the installation guide" to distinguish upfront that these instructions are not applicable.

I have just moved the RunCode instructions to one place so this is no longer necessary. I now just point the readers to the RunCode section.